docs/mobile-apps/features/mobile-app-diagnostics/app-crash-logs.md

```diff
@@ -39,15 +66,18 @@ You can download the crash log as a text file to save and share with Sauce Labs
 
 :::note
 You can also download the results using the API call below and send it to your development team:
+
 ```java
 curl --compressed \
 -O https://{SAUCE_USERNAME}:{SAUCE_ACCESS_KEY}@{DATA_CENTER}.saucelabs.com/v1/rdc/jobs/{JOB_ID}/crash.log`
 ```
+
 :::
 
 ## Limitations
 
 :::note
+
 - Does not capture hangs and cross-platform development frameworks.
 - Emulators/Simulators are not supported.
 
```

Comment:
We should also note that this will override any native crash reporting that they currently have in place so they shouldn't expect to see the crash report to show up in their own system if they have something else integrated (including if Backtrace is already integrated).